exporters/jaeger-proto/build.gradle.kts

```diff
@@ -0,0 +1,17 @@
+plugins {
+  id("otel.protobuf-conventions")
+  id("otel.publish-conventions")
```

Comment:
We ourselves were using protos in another module, the sdk-autoconfigure tests. So I realized that the best we can do is to separate out this artifact. We could still make a call to remove this line until we get a complaint, though keeping this and calling out in release notes that users should add this artifact, would allow still maintaining semconv without worries.

Reply:
Do you mind adding a bit to the pending CHANGELOG for this as a part of this PR, so we don't forget? Thanks!

Reply:
Sure added